src/server/kiwix-serve.cpp

```diff
@@ -324,17 +324,13 @@ int main(int argc, char** argv)
   auto libraryFileTimestamp = newestFileTimestamp(libraryPaths);
   auto curLibraryFileTimestamp = libraryFileTimestamp;
 
-  /* Infer ipMode from address */
-  kiwix::IpMode ipMode = kiwix::IpMode::ipv4;
-
-  if (address == "all"){
-    address = "";
-    ipMode = kiwix::IpMode::all;
-  } else if (address == "ipv4"){
-    address = "";
-  } else if (address == "ipv6"){
-    address = "";
-    ipMode = kiwix::IpMode::ipv6;
+  kiwix::IpMode ipMode =  (address == "all")  ? kiwix::IpMode::ALL  :
+                          (address == "ipv6") ? kiwix::IpMode::IPV6 :
+                          (address == "ipv4") ? kiwix::IpMode::IPV4 :
+                                                kiwix::IpMode::FLEX;
```

Comment:
Why do we still have `FLEX`? The purpose of capitalizing the `IpMode` enum members was so that we could use the more intuitive word `AUTO`.

Reply:
Sorry for this. Missed this point after coming back. Will fix on next iteration.